drivers/scheduler/k8s/k8s.go

```diff
@@ -616,6 +616,72 @@ func (k *k8s) GetNodesForApp(ctx *scheduler.Context) ([]node.Node, error) {
 	return result, nil
 }
 
+func (k *k8s) GenerateSupportBundle(ctx *scheduler.Context) (string, error) {
+	var output string
+	var err error
+	for _, spec := range ctx.App.SpecList {
+		if obj, ok := spec.(*apps_api.Deployment); ok {
+			output += insertLineBreak(obj.Name)
+			var depStatus apps_api.DeploymentStatus
+			if depStatus, err = k8sutils.DescribeDeployment(obj); err != nil {
+				output += fmt.Sprintf("%v", &ErrFailedToValidateApp{
```

Comment:
Why is `GenerateSupportBundle` returning error type for validating app? It should have it's own error type.

Reply:
I was thinking of embedding the root error here but ended up neither here nor there. Added a new error type. ErrFailedToGetAppStatus.